src/main/java/io/vertx/circuitbreaker/CircuitBreakerOptions.java

```diff
@@ -139,6 +147,12 @@ public class CircuitBreakerOptions {
    */
   private long failuresRollingWindow = DEFAULT_FAILURES_ROLLING_WINDOW;
 
+
+  /**
+   * The way we determine if a future is indeed a failure.
+   */
+  private Predicate<AsyncResult> asyncResultTest = DEFAULT_ASYNC_RESULT_FAILURE_TEST;
```

Comment:
I think Failsafe's name for this (a failure policy) says more than async result test. Would you mind changing it?

Reply:
And updating related names and Javadoc in this class